l10n_br_fiscal/models/data_ncm_nbs_abstract.py

```diff
@@ -33,45 +33,36 @@ class DataNcmNbsAbstract(models.AbstractModel):
 
     estimate_tax_national = fields.Float(
         string="Estimate Tax Nacional Percent",
+        company_dependent=True,
         store=True,
-        readonly=True,
         digits="Fiscal Tax Percent",
-        compute="_compute_amount",
     )
 
     estimate_tax_imported = fields.Float(
         string="Estimate Tax Imported Percent",
+        company_dependent=True,
         store=True,
-        readonly=True,
         digits="Fiscal Tax Percent",
-        compute="_compute_amount",
     )
 
-    @api.depends("tax_estimate_ids")
-    def _compute_amount(self):
-        for record in self:
-            object_field = OBJECT_FIELDS.get(record._name)
-            last_estimated = record.env["l10n_br_fiscal.tax.estimate"].search(
-                [
-                    (object_field, "=", record.id),
-                    ("company_id", "=", record.env.company.id),
-                ],
-                order="create_date DESC",
-                limit=1,
-            )
-
-            if last_estimated:
-                record.estimate_tax_imported = (
-                    last_estimated.federal_taxes_import
-                    + last_estimated.state_taxes
-                    + last_estimated.municipal_taxes
-                )
-
-                record.estimate_tax_national = (
-                    last_estimated.federal_taxes_national
-                    + last_estimated.state_taxes
-                    + last_estimated.municipal_taxes
-                )
+    def update_compute_amount(self, last_estimated):
```

Comment:
Aqui o nome do método não seria mais claro assim?

```suggestion
    def update_estimated_taxes(self, last_estimated):
```